osu.Game/Screens/OnlinePlay/Multiplayer/Match/MatchStartControl.cs

```diff
@@ -214,8 +206,7 @@ private void updateState()
 
             readyButton.Enabled.Value = countdownButton.Enabled.Value =
                 client.Room.State != MultiplayerRoomState.Closed
-                && SelectedItem.Value?.ID == client.Room.Settings.PlaylistItemId
```

Comment:
https://github.com/ppy/osu/pull/32250/files#r1998394095

> Why is this check gone?

This is more appropriate to be answered in the context of https://github.com/ppy/osu/pull/32250 as a whole, rather than the current state of multiplayer....

The screen itself shall always ensure it is applying the room's currently selected item at all times. If it needs to do any validation, it would do so at the point of starting play. It is not the responsibility of the ready button to determine whether the screen is in a good state, and _nor can it_ - this bindable being in the correct value has no indication as to whether the beatmap is set appropriately, whether all schedules have run, etc.

Reply:
Can probably agree with this one.